File: PAWIM_LAB_5/web/src/main/java/com/example/webclient/model/Ram.java
package com.example.webclient.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Ram {
    private String name;
    private Integer ramCapacity;
    private Integer memoryRate;
}

File: PAWIM_LAB_7/P12MAUI.Client/Model/Ram.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace P12MAUI.Client.Model
{
    public class Ram
    {
        public string Name { get; set; }
        public int RamCapacity { get; set; }
        public int MemoryRate { get; set; }

        // Konstruktor
        public Ram(string name, int ramCapacity, int memoryRate)
        {
            this.Name = name;
            this.RamCapacity = ramCapacity;
            this.MemoryRate = memoryRate;
        }
    }
}
